dropwizard-demo/src/main/java/com/dep/resource/Controller.java

```diff
@@ -1,80 +1,74 @@
 package com.dep.resource;
 
 import com.dep.Services.TwitterImplement;
+import com.dep.models.SendResponse;
 import com.dep.models.TweetResponse;
 import org.eclipse.jetty.util.StringUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 import twitter4j.Status;
 import twitter4j.TwitterException;
 
-import javax.ws.rs.*;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import java.util.Date;
 import java.util.List;
 
-
 @Produces(MediaType.APPLICATION_JSON)
-@Path("/api/1.0/twitter")
+@RestController
 public class Controller {
 
     public static final Logger logger = LoggerFactory.getLogger(Controller.class);
+    @Autowired
     TwitterImplement twitterImplement;
 
-    public Controller(TwitterImplement twitterImplement) {
-        this.twitterImplement = twitterImplement;
-    }
 
-    public Controller() {
-        twitterImplement = new TwitterImplement();
-    }
-
-    @GET
-    @Path("/getTweets")
+    @GetMapping("/getTweets")
     public Response fetchTweets() {
         return Response.ok(twitterImplement.fetchLatestTweets()).build();
     }
 
-    @GET
-    @Path("/filteredTweets")
+
+    @RequestMapping("/filteredTweets/{searchKey}")
```

Comment:
only path is mentioned, specifically mentioning HTTP method will make the API more robust

Reply:
This is still not addresses